### PR TITLE
CS-44350 - Added support for removing the branches that are not being exported in the workflows

### PR DESCRIPTION
**Pull request by cs-raj**
Added the check to remove all the other branches than the one mentioned in the import and while doing audit we can pass branch to be kept using config flag and version bump currently only for workflows

version bump:
@contentstack/cli-audit: 1.5.0 -> 1.5.1

### CODE REVIEW

File: packages/contentstack-audit/src/audit-base-command.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export abstract class AuditBaseCommand extends BaseCommand<typeof AuditBaseComma
         config: this.sharedConfig,
         fix: this.currentCommand === 'cm:stacks:audit:fix',
       };
-
+      // console.log(this.sharedConfig);
```
Comment (aman19K) [inline packages/contentstack-audit/src/audit-base-command.ts:116]: @cs-raj , pls remove unused console logs

**Review by aman19K — APPROVED** [review]

File: packages/contentstack-audit/src/types/content-types.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ type RefErrorReturnType = {
   uid?: string;
   content_types?: string[];
   title?: string;
+  branches?: string[];
```
Comment (antonyagustine) [inline packages/contentstack-audit/src/types/content-types.ts:54]: Can you remove the `branches` key from here? And create a new type and extend `branches` and other keys it there?

File: packages/contentstack-audit/src/types/content-types.ts [hunk 3]
```diff
@@ -119,6 +120,7 @@ enum OutputColumn {
   title = 'title',
   'uid' = 'uid',
   'missingCts' = 'content_types',
+  'Missing Branches' = 'branches',
```
Comment (antonyagustine) [inline packages/contentstack-audit/src/types/content-types.ts:123]: Please fix this as well.!

**Review by antonyagustine — CHANGES_REQUESTED** [review]

File: packages/contentstack-audit/src/audit-base-command.ts (hunk 1 above)
Comment (cs-raj) [reply packages/contentstack-audit/src/audit-base-command.ts]: Done

File: packages/contentstack-audit/src/types/content-types.ts (hunk 2 above)
Comment (cs-raj) [reply packages/contentstack-audit/src/types/content-types.ts]: Removed the branches

File: packages/contentstack-audit/src/types/content-types.ts (hunk 3 above)
Comment (cs-raj) [reply packages/contentstack-audit/src/types/content-types.ts:124]: This one cannot be done as it's commonly used while creating the CSV file

**Review by aman19K — APPROVED** [review]

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**